R/get_latest_resource.R

```diff
@@ -0,0 +1,105 @@
+#' Get the latest resource from a data set
+#'
+#' Returns the latest resource available in a dataset.
+#'
+#' There are some datasets on the open data platform that
+#' keep historic resources instead of updating existing ones.
+#' For these it is useful to be able to retrieve the latest
+#' resource. As of 5.7.2024 these data sets include:
+#' * gp-practice-populations
+#' * gp-practice-contact-details-and-list-sizes
+#' * nhsscotland-payments-to-general-practice
+#' * dental-practices-and-patient-registrations
+#' * general-practitioner-contact-details
+#' * prescribed-dispensed
+#' * prescriptions-in-the-community
+#' * community-pharmacy-contractor-activity
+#'
```

Comment:
prescriptions-in-the-community fails to identify latest id, it looks like new breakdowns were added to this dataset so no longer applicable (unless the new breakdowns would be moved to a separate dataset)
dispenser-location-contact-details should be added to the list